src/mapclient/core/managers/optionsmanager.py

```diff
@@ -16,12 +16,16 @@ def _is_boolean(option):
                       USE_EXTERNAL_GIT, USE_EXTERNAL_RCC, USE_EXTERNAL_UIC, INTERNAL_WORKFLOWS_AVAILABLE, AUTOLOAD_PREVIOUS_WORKFLOW]
 
 
+def _is_float(option):
+    return option in [CLOSE_AFTER]
+
+
 class OptionsManager(object):
 
     def __init__(self):
         # Set default values
         self._options = {
-            SHOW_STEP_NAMES: True, DONT_CREATE_VIRTUAL_ENV: False, CHECK_TOOLS_ON_STARTUP: True,
+            SHOW_STEP_NAMES: True, CLOSE_AFTER: 2000, DONT_CREATE_VIRTUAL_ENV: False, CHECK_TOOLS_ON_STARTUP: True,
```

Comment:
You are expecting the default to be 2000 seconds?

Reply:
Typo. My bad.